.github/workflows/repository_lastet_added_files.yml

```diff
@@ -0,0 +1,41 @@
+name: . 🔍🔍 Show N files added before and after datetime into the S3 repository
+
+on:
+  workflow_dispatch:
+    inputs:
+      count:
+        description: 'Amount of files to show'
+        type: string
+        default: '500'
+        required: true
+      date_time:
+        description: 'Date and time to show files added before and after (YYYY-MM-DD HH:MM:SS)'
```

Comment:
***nit***: Would it be required to know that it is in `UTC` timezone? Maybe the description can mention that.